vscode/package.json

```diff
@@ -329,10 +329,16 @@
 		"webpack-cli": "^4.7.2"
 	},
 	"dependencies": {
+		"@vscode/test-electron": "^2.1.5",
```

Comment:
`vscode-test` was renamed to `@vscode/test-electron`.
This was the only manual change (here and at its use site) after running `npm update`.